docs/src/main/sphinx/security/cert-auth.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+Trino can be configured to support certified clients seeking validation from a
```

Comment:
I'm not super thrilled with "can be" - what about "Configure Trino to support certified clients that seek validation from a Trino server." or "You can configure Trino to support..."

Reply:
Yeah, the Passive Monstre took over my fingers. I beat it back down. Thanks.